pipeline_tools/input_utils.py

```diff
@@ -21,6 +21,27 @@ def get_sample_id(bundle):
     return sample_id
 
 
+def get_expected_cell_count(bundle):
+    """Return the total estimated cells from the given bundle, otherwise use the same default value as CellRanger
+    (3000 cells): https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count
+
+    Args:
+        bundle (humancellatlas.data.metadata.Bundle): A Bundle object contains all of the necessary information.
+
+    Returns:
+        total_estimated_cells (str): Int giving the total number of estimated cells
+    """
+    cell_suspension = [f for f in bundle.biomaterials.values() if isinstance(f, CellSuspension)]
+    n_cell_suspension = len(cell_suspension)
+    if n_cell_suspension != 1:
+        raise ValueError('The data bundle should contain exactly 1 cell_suspension.json file, not {}'.format(n_cell_suspension))
+    total_estimated_cells = cell_suspension[0].total_estimated_cells
```

Comment:
nit: `total_estimated_cells = int(cell_suspension[0].total_estimated_cells)`

Reply:
Good suggestions! `total_estimated_cells` is an optional field and if it is not there the metadata api library returns `None`, so I did `int(total_estimated_cells) if total_estimated_cells else default_estimated_cells` below